docs/govuk-verify.md

```diff
@@ -35,53 +35,191 @@ curl localhost:50300/admin/healthcheck
+##### Enrol
+
+Complete these steps for both signing and encryption certificates.
+
+Open the following document:
+
+https://docs.google.com/document/d/1lH4b2uOn35rdzLAzW5I-_O9wG9NU8fTre8FaM0Y97L8/edit?pli=1#
+
+And go to the appropriate url for the environment you are enrolling the
+certificates for, either integration or production.
+
+- Choose ENROL
+- Upload the CSR file
+- Enter the details in the form:
+  - Applicant Details: your name and the group email address
+    (capt-dev@digital.education.gov.uk), ensue the email address is correct as
+    this is where the certificate will be sent.
+  - Certificate Profile: Select the type of certificate either SAML Signing or
+    SAML Encryption, ensure this matches the CSR you are uploading
+  - Challenge Phrase: the challenge phrase used above
+
+Once submitted you will receive a confirmation email, the certificates will be
+generated within 5 days and we be sent to the group email address.
 
-#### Importing certificates
+##### Import
 
-Once we get certificates back from Verify, we want to store them in the Key
-Vault alongside the keys and requests. Run the following to automate that
-process.
+When you receive the certificates run the import script for both signing and
+encryption certificates:
 
 ```bash
 bin/import-vsp-certs <environment> <version>
 ```
 
-Where `<environment>` and `<version>` match the values used to generate the
-certificate request above.
+Using the same environment and version as you set in the `request-vsp-certs`
+script.
+
+When prompted, paste the certificate body for the type of certificate (signing
+or encryption) - be careful to note which type of certificate you are importing.
+
+The script copies the certificates to the Azure Keyvault.
+
+##### Rotate
+
+The steps to actually rotate the keys varies between the two types of
+certificate
+
+[Verify docs for this step](https://www.docs.verify.service.gov.uk/maintain-your-connection/rotate-keys/#rotate-your-encryption-and-signing-keys)
+
+###### Signing
+
+Now that the signing key is stored in the Azure Keyvault, the next step gets the
+application to use it:
+
+- Send the GOVUK Verify team an email to
+  idasupport+onboarding@digital.cabinet-office.gov.uk with the signing
+  certificate letting them know you are rotating the signing key
+
+Once you have a response that the certificate is deployed, move on
+
+- Locate the `production.template.json` and/or `development.templtate.json` in
+  `azure/app/parameters` for the environment you are updating
+- Open the file and edit the `VSP.SAML_SIGNING_KEY` entry
+- You'll see the `secret_name` matches the one in the Azure Keyvault with the
+  old version number
+- Update the version number, commit the changes and open a PR to get them merged
+- The next deploy will update the keys, if updating production you will have to
+  perform a release
+- Let the GOVUK Verify team know the signing key has been rotated, they will
+  delete the old key
+
+###### Encryption
```

Comment:
```suggestion
###### Encryption key
```

It think it would be good to be clear that this isn't an action "we need to encrypt something" and is a label of the type of key.